tbx/blog/models.py

```diff
@@ -130,10 +140,18 @@ def set_body_word_count(self):
         ).split()
         self.body_word_count = len(body_words)
 
+    @cached_property
+    def sectors(self):
+        return self.related_sectors.all()
+
     @cached_property
     def services(self):
         return self.related_services.all()
 
+    @property
+    def tags(self):
+        return chain(self.services, self.sectors)
```

Comment:
I notice that here you have used `chain` which I think just appends the services onto the sectors, whereas on the blog index page you used `union`, which appears to intermix them (I'm not sure what logic it is using for ordering). I suspect we want `chain` for both, but let's check in at stand-up.